cmd/snap/cmd_snap_op.go

```diff
@@ -183,24 +183,51 @@ type cmdRefresh struct {
 	Channel    string `long:"channel" description:"Refresh to the latest on this channel, and track this channel henceforth"`
 	Positional struct {
 		Snap string `positional-arg-name:"<snap>"`
-	} `positional-args:"yes" required:"yes"`
+	} `positional-args:"yes"`
 }
 
-func (x *cmdRefresh) Execute([]string) error {
+func refreshAll() error {
 	cli := Client()
-	name := x.Positional.Snap
-	opts := &client.SnapOptions{Channel: x.Channel}
-	changeID, err := cli.Refresh(name, opts)
+	updates, err := cli.List(&client.ListOptions{RefreshOnly: true})
```

Comment:
TBH I started with this in the client because it was more convenient. It seemed like it made implmenting the wait() simpler. However with a fresh (and less jet-lagged) brain I realize that this is probably not actually true. I think a refreshAll() in the server that just returns a big taskset with all the updates would work just fine for the client as well. I can create a followup branch based on this.